File: 오선민/programmers/P181909.py
'''
문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181909
시간: 0.04ms
메모리: 10.3MB
PR 제목 : 날짜 사이트_명 문제_번호 사용_언어
'''

def solution(my_string):
    return sorted([my_string[i::] for i in range(len(my_string))])

File: 공예진/programmers/P181909.java
//문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181910
//시간: 0.38ms
//메모리: 69.6MB

import java.util.*;

class Solution {
    public String[] solution(String my_string) {
        String[] answer = new String[my_string.length()];

        for (int i = 0; i < my_string.length(); i++) {
            answer[i] = my_string.substring(i);
        }

        Arrays.sort(answer);
        return answer;
    }
}